src/Symfony/Bundle/SecurityBundle/DependencyInjection/Compiler/AddSecurityVotersPass.php

```diff
@@ -41,16 +43,33 @@ public function process(ContainerBuilder $container)
             throw new LogicException('No security voters found. You need to tag at least one with "security.voter".');
         }
 
+        $debug = $container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug');
```

Comment:
I'm not sure it's the best approach. Is it better to test `kernel.debug` here, or create a specific compiler pass only added if `kernel.debug` is set to true ?

Reply:
ok as is to me

Reply:
@chalasr ok, thanks for your help

Reply:
but we IMO don't need the `hasParameter()` call here

Reply:
I did it in this way because it was like that in `SecurityExtension.php`. So I modified it in `SecurityExtension.php` and `AddSecurityVoterPass.php` and fixed the tests that change broke.